includes/free-trial/class-wc-calypso-bridge-free-trial-product-form-changes.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+class WC_Calypso_Bridge_Free_Trial_Product_Form_Changes {
+	/**
+	 * The single instance of the class.
+	 *
+	 * @var object
+	 */
+	protected static $instance = null;
+
+	/**
+	 * Get class instance.
+	 *
+	 * @return object Instance.
+	 */
+	final public static function get_instance() {
+		if ( is_null( self::$instance ) ) {
+			self::$instance = new self();
+		}
+
+		return self::$instance;
+	}
+
+	public function __construct() {
+		// Remove pinterest tab.
+		add_filter( 'woocommerce_product_data_tabs', [ $this, 'remove_pinterest' ], 99);
+
+		// remove "Sharing" meta box.
+		add_filter( 'sharing_meta_box_show', '__return_false' );
+
+		// Customize Channel visibility meta box.
+		add_action( 'admin_enqueue_scripts', [ $this, 'add_scripts_for_google_listing_ads' ], 10, 10);
+	}
+
+	public function remove_pinterest( $tabs ) {
+		if ( isset( $tabs[ 'pinterest_attributes' ] ) ) {
+			unset( $tabs[ 'pinterest_attributes' ] );
+		}
+		return $tabs;
+	}
+
+	public function add_scripts_for_google_listing_ads( $hook ) {
+	    global $post;
```

Comment:
A tiny thing, this looks like at a different indentation level. 🙂

Reply:
@chihsuan Thank you! I'll try to look into it. My Phpstorm has done this a few times now 😞